src/global_logic.js

```diff
@@ -1792,6 +1796,9 @@ function getCharaLB(chara) {
         "ATK": 0,
         "HP": 0,
         "Element": 0,
+        "OugiDamage": 0,
```

Comment:
Currently these three keys mapped as String objects in loop at line 1817. While it works, in order to have consistency these should be mapped into numbers as all other fields if we evaluate them as such
if (key == "ATK" || key == "HP" || key == "Element") {
                LB[key] = parseInt(chara[exactKey]);
            }

Reply:
done.